### ISSUE TITLE
Support coverage reporting in Azure DevOps

### ISSUE THREAD

**Issue by pascalberger** [cake-contrib/Cake.Recipe#529]
Azure DevOps supports code coverage metrics (see [Code coverage metrics and branch policy for pull requests](https://docs.microsoft.com/en-us/azure/devops/server/release-notes/azuredevops2020?view=azure-devops&branch=releasenotes%2FAzureDevOpsServer2020#code-coverage-metrics-and-branch-policy-for-pull-requests)). It would be great if Cake.Recipe would automatically send coverage data while running under Azure DevOps. See [Surface test results in the Tests tab](https://docs.microsoft.com/en-us/azure/devops/pipelines/test/review-continuous-test-results-after-build?view=azure-devops#surface-test-results-in-the-tests-tab) for possible integration points.

To use the API the required functionality to upload coverage data can be added to [Cake.AzureDevOps](https://github.com/cake-contrib/Cake.AzureDevOps).

**Comment by gep13** [comment]
@pascalberger I would be happy to see this getting added.  Is this something that I can assign to you?

**Comment by pascalberger** [comment]
Yes, I can look at it (but some other issues which I need to tackle first)

**Comment by gep13** [comment]
@pascalberger i am starting to think about getting ready to ship 2.0.0, and as such, I am going to move this one from the milestone.  If you get to it before the release, then I would be happy to bring this in.
